blogposts/2021/release-3.34

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sourcegraph 3.34 release"
+publishDate: 2021-11-23T10:00-07:00
+description: "Sourcegraph 3.34 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.34"
+published: false
+heroImage:
+socialImage:
+changelogItems:
+
+---
+
+Sourcegraph 3.34 is now available! For this release, we introduced:
+
+## No results page
+When your search has no results, we will now display helpful tips on how to use Sourcegraph search, examples of how to solve common query issuses, and links to help videos addressing common search use cases.
```

Comment:
```suggestion
When your search comes back with no results it can sometimes be hard to know why. We now display helpful tips on how to use search, examples of how to solve common query issues, and links to help videos addressing common use cases.
```